diff --git a/data/constants.py b/data/constants.py
--- a/data/constants.py
+++ b/data/constants.py
@@ -1,3 +1,3 @@
 # LINT.IfChange
-MIN_VERSION = "0.2.0"
+MIN_VERSION = "0.3.0"
 # LINT.ThenChange(constants.rs)
diff --git a/data/constants.rs b/data/constants.rs
--- a/data/constants.rs
+++ b/data/constants.rs
@@ -1,3 +1,3 @@
 // LINT.IfChange
-const MIN_VERSION: &str = "0.2.0";
+const MIN_VERSION: &str = "0.3.0";
 // LINT.ThenChange(constants.py)
